shared_utils/corpora.py

```diff
@@ -7,10 +7,30 @@
 """
 import collections
 import json
+import re
 
-from server.cli.upgrade import validate_version_str
-from server.common.utils.corpora_constants import CorporaConstants
+from shared_utils.utils.corpora_constants import CorporaConstants
 
+# Official SemVer regex: https://semver.org/
```

Comment:
I think this chunk of code is specifically for the desktop version only since we use it to bump the PyPi release so I don't think it makes sense to move it to shared.